### DISCUSSION TITLE
which map file formats are currently supported

### DISCUSSION THREAD

**Discussion by CodeK1988** [WorldWindEarth/WorldWindKotlin#45] [Q&A]
hi mate, which map file formats are currently supported. except gpkg?

**Comment by EMaksymenko** [comment]
If we are speaking about map cache only (not about online maps), then you
can easily organize pure file cache with images in folders. No other SQLite
based formats like MBTiles or other are supported. Only GPKG.

чт, 16 лист. 2023, 11:20 користувач Code K ***@***.***> пише:

**Reply by CodeK1988** [reply]
will other formats be supported in the future?  I have offline map files in mbtiles format, and dn’t know how to convert them to GPKG format.by the way, thank you for your reply!

**Reply by EMaksymenko** [reply]
We do not plan to support other cache file formats.

You can convert it using some GIS software like GlobalMapper, ArcGIS etc...
